tests/unit/statistical_testing/QuadraticTimeMMD_unittest.cc

```diff
@@ -354,7 +354,7 @@ TEST(QuadraticTimeMMD, perform_test_permutation_biased_full)
 	// assert against local machine computed result
 	mmd->set_statistic_type(ST_BIASED_FULL);
 	float64_t p_value=mmd->compute_p_value(mmd->compute_statistic());
-	EXPECT_NEAR(p_value, 0.0, 1E-10);
+	EXPECT_NEAR(p_value, 0.8, 1E-10);
```

Comment:
@karlnapf I think the following tests follow similar pattern:

TEST(QuadraticTimeMMD, perform_test_permutation_biased_full)
TEST(QuadraticTimeMMD, perform_test_permutation_unbiased_full)
TEST(QuadraticTimeMMD, perform_test_permutation_unbiased_incomplete)
TEST(QuadraticTimeMMD, perform_test_spectrum)
TEST(LinearTimeMMD, perform_test_gaussian_biased_full)
TEST(LinearTimeMMD, perform_test_gaussian_unbiased_full)
TEST(LinearTimeMMD, perform_test_gaussian_unbiased_incomplete)
TEST(KernelSelectionMaxMMD, linear_time_single_kernel_streaming)
TEST(KernelSelectionMaxMMD, quadratic_time_single_kernel_dense)
TEST(KernelSelectionMaxMMD, linear_time_weighted_kernel_streaming)
TEST(KernelSelectionMaxTestPower, linear_time_single_kernel_streaming)
TEST(KernelSelectionMaxTestPower, quadratic_time_single_kernel)
TEST(KernelSelectionMaxTestPower, linear_time_weighted_kernel_streaming)
TEST(KernelSelectionMaxCrossValidation, quadratic_time_single_kernel_dense)
TEST(KernelSelectionMaxCrossValidation, linear_time_single_kernel_dense)
TEST(KernelSelectionMedianHeuristic, quadratic_time_single_kernel_dense)
TEST(KernelSelectionMedianHeuristic, linear_time_single_kernel_dense)

However, I think instead of dropping all of these, some of these could be rewritten with hard coded data and asserted accordingly. Let me know what you think.